icon/Icon.ts

```diff
@@ -292,6 +281,46 @@ export const Icon = {
     x(p?: IconProps)                {return Icon.icon({...p,  iconName: 'times'})},
     xCircle(p?: IconProps)          {return Icon.icon({...p,  iconName: 'times-circle'})},
 
+    /**
+     * Allows apps to override Hoist choices for given icons.
+     */
+    //Icons for column sorting
+    sortAsc: () => Icon.arrowUp(),
```

Comment:
why can't these get props?  Also we are adding an extra wrapping layer.  

I think we should `apply` all these aliases to the object after it is constructed, and then it can be just a super simple alias -- `sortAbsAsc: Icon.arrowToTop`

Reply:
And now, thinking about it, wonder if it would actually just be more straightforward to have *all* of the functions going directly to the raw Icon.icon call.   

The distinction between `Alias` and the core functions just becoming a bit thin.

So essentially this project is just about adding some new  functions that represent higher-level UI operations.

Reply:
@Ryanseanlee / @lbwexler - sounds like we might want to check in together and confirm what Lee is saying here to better standardize these aliases and maybe cut down on some boilerplate wrapping that's repeated again and again. Not looking at the code directly at this moment, it's not 100% clear what the cleanest/most elegant approach would be here, but I am sure we could decide on it quickly if we touch base together today.  

Would love to get this wrapped up.

Reply:
Would love to sync up and get this tightened up.

Reply:
When applying the aliases to the Icon object after construction, Typescript doesn't like when the object shape changes (i.e., when applying the aliases after construction). One way is to type the `Icon` object as `PlainObject,` although we lose the TS help. Thinking the best solution is to still just apply the aliasing in the construction of the object